src/vega.schema.ts

```diff
@@ -67,16 +82,23 @@ export type VgScale = {
   zero?: boolean
 }
 
-export function isUnionedDomain(domain: any[] | UnionedDomain | VgDataRef): domain is UnionedDomain {
+export function isDataRefUnionedDomainDomain(domain: VgDomain): domain is DataRefUnionedDomain {
```

Comment:
isFieldRefUnionDomainDomain

why repeated `DomainDomain`?  It reads pretty confusing.  One `Domain` seems enough. 

Also why `Union`, not `Unioned` like below?